index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Stanley Personal Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_bar heading heading-pl">
+        <a href="" class="logo_info">Stanley</a>
+        <button class="logo_vector-btn">
+          <img src="images/Icon - Menu.png" alt="Vector" />
+        </button>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="welcome">
+        <div class="container-1">
+          <h1 class="title">I am Stanley, glad to see you!</h1>
+          <p class="detailed">
+            I am a software developer! I am capable of building software
+            products including features and websites. Please go through my work
+            and experience! In case you are impressed by what you have seen, do
+            feel free to contact me.
+          </p>
+          <h2 class="connect">LET'S CONNECT</h2>
```

Comment:
- [ ] Once again, you did a really great job with the semantics here using `<main>` and `<section>` tags, but there are some issues with your design as shown below:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/205152664-80781d7c-e013-4af0-a6a6-d883c0cc0131.png) | ![image](https://user-images.githubusercontent.com/77069011/205152600-9fc4ca29-c5d2-4d5a-85ec-9ba8c71f9f1a.png) |
 
Kindly notice that there are some visual mismatches, such as the background image not being correctly implemented, the main title needing an increase in its size, the paragraph doesn't have the correct font family, the "Let's Connect" subtitle needing a decrease in its size, as well as the correct color and alignment, and, lastly, the social media icons needing the correct alignment to the left. 🎨 

Please consider implementing these changes as they will improve tremendously the quality of your portfolio! 🤓 🛠️